modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc

```diff
@@ -0,0 +1,15 @@
+:description: This page describes how to delete composite databases.
+[role=enterprise-edition not-on-aura]
+[[composite-databases-delete]]
+= Delete composite databases
+
+Composite databases can be deleted using the command `DROP COMPOSITE DATABASE name`.
+This command will fail if the targeted database is not a composite database.
```

Comment:
```suggestion
However, keep in mind that the first command will fail if the target database is not composite, while the second one will not fail because it targets any database.
```